onnx2tf/onnx2tf.py

```diff
@@ -73,6 +73,8 @@ def convert(
     quant_type: Optional[str] = 'per-channel',
     custom_input_op_name_np_data_path: Optional[List] = None,
     input_output_quant_dtype: Optional[str] = 'int8',
```

Comment:
If you are separating the input and output quantization types, the input_output_quant_dtype function is unnecessary and should be removed from the API.

Reply:
Yes I thought so, this was a quick fix to test it. I'll clean it up